### PR TITLE
Add some commentary around Network Sizing and the math behind it

### PR DESCRIPTION
**Pull request by mwitzenm**
#### What is this PR About?
Add some commentary around the Network Sizing a the host length bits

#### How should we test or review this PR?
Review language

#### Is there a relevant Trello card or Github issue open for this?
https://trello.com/c/VddVZeP9/108-introduction-and-fundamentals-on-the-sdn-cidr-and-sizing-for-openshift
#### Who would you like to review this?
cc: @redhat-cop/cant-contain-this

### CODE REVIEW

File: playbooks/installation/index.adoc [hunk 1]
```diff
@@ -95,6 +95,11 @@ Each service in the cluster will be assigned an IP from this range.  The default
 
 Both the openshift_portal_net and the osm_cluster_network_cidr will default to the value above if not set, however if you prefer to have your inventory file describe your environment, its recommended to set them explicitly for future reference or comparison.
 
+==== Host Subnet Length
+*Ansible Host File Variable:* `osm_host_subnet_length=9`
```
Comment (etsauer) [inline playbooks/installation/index.adoc]: If we're going to list the variable, we should also add it to our host file samples.

File: playbooks/installation/index.adoc [hunk 2]
```diff
@@ -95,6 +95,11 @@ Each service in the cluster will be assigned an IP from this range.  The default
...
+
+Each OpenShift node will require its own subnet which allows the pods to obtain an IP in that range.  This variable in the ansible hosts file sets that subnet.  The 9 in the example above represents a /23 subnet or 2 class C subnets or also described as 510 usable IP addresses.  What does this have to do with the /14 cluster network mentioned above.  It all boils down to a network math problem.  We have a total number of IP addresses to use that is established by the /14 refereced above.  So now to the math:  262,142 (total IP) / 510 (total IP per node) = ~514 total nodes in your cluster that will run a total of 510 pods on each node in your cluster.
```
Comment (etsauer) [inline playbooks/installation/index.adoc]: Missing a question mark.

Comment (etsauer) [inline playbooks/installation/index.adoc]: I think the math equiatiot should break out into its own paragraph, and be bolded and italicized.

File: playbooks/installation/index.adoc [hunk 3]
```diff
@@ -120,13 +139,14 @@ openshift_master_api_port=443 <1>
 openshift_master_console_port=443 <2>
 openshift_portal_net=172.30.0.0/16 <3>
 osm_cluster_network_cidr=10.128.0.0/14 <4>
+osm_host_subnet_length=9 <5>
```
Comment (etsauer) [inline playbooks/installation/index.adoc]: Need to add this to all hostfile snippets in the doc so we don't lose it after this section.

Comment (mwitzenm) [reply playbooks/installation/index.adoc]: I have added all this, let me know if there are other outstanding issues

**Pull request merged**